src/components/CreateCourseRunPage/CreateCourseRunForm.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import RenderInputTextField from '../RenderInputTextField';
+
+const BaseCreateCourseRunForm = ({
+  handleSubmit,
+  pristine,
+  submitting,
+  title,
+  uuid,
+}) => {
+  const formControlDisabled = pristine || submitting;
+  return (
+    <React.Fragment>
+      <div className="create-course-run-form row">
+        <div className="col">
+          <h2>Create a new course run</h2>
+          <hr />
+          <div className="row">
+            <div id="course" className="col-12">
+              <strong>Course</strong>
+            </div>
+            <div className="col-12">
+              <Link to={`/courses/${uuid}/edit/`}>
+                {title}
+              </Link>
+            </div>
+          </div>
+          <form onSubmit={handleSubmit}>
+            <Field
+              name="start"
+              type="date"
+              component={RenderInputTextField}
+              label={
+                <React.Fragment>
+                  <strong>Start date</strong>
+                  <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name="end"
+              type="date"
+              component={RenderInputTextField}
+              label={
+                <React.Fragment>
+                  <strong>End date</strong>
+                  <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <div className="row justify-content-end">
+              <Link to={`/courses/${uuid}/edit/`}>
+                <button
+                  className="btn btn-outline-primary"
+                  disabled={submitting}
+                >
+                  Cancel
+                </button>
+              </Link>
+              <button
+                type="submit"
+                className="btn btn-outline-primary form-submit-btn"
+                disabled={formControlDisabled}
+              >
+                Submit
+              </button>
+
+            </div>
+          </form>
+        </div>
+      </div>
+    </React.Fragment>
+  );
+};
+
+BaseCreateCourseRunForm.defaultProps = {
+};
+
+BaseCreateCourseRunForm.propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  initialValues: PropTypes.shape({ // eslint-disable-line react/no-unused-prop-types
+  }).isRequired,
```

Comment:
You coud do `shape().isRequired` I think - might avoid that eslint warning?

Reply:
I will give this a try.

Reply:
I actually ended up removing it.  I realized I Was not using it @mikix

Reply:
Heh, also a good solution. :)